src/lang/cpp/internal/BindingsResolver.java

```diff
@@ -137,27 +141,49 @@ private void err(String msg) {
 //		stdErr.println(spaces() + msg.replace("\n", "\n" + spaces()));
 	}
 
+	/**
+	 * This method can only be called once.
```

Comment:
Now that this class holds state, it should be resettable (cf. `Parser::reset()`)